mojave/effects/spawners/lootdrop/armorloot.dm

```diff
@@ -247,7 +247,7 @@
 				)
 	var/loot5 = list(
 				/obj/item/clothing/suit/armor/ms13/vest/military,
-				/obj/item/clothing/head/ms13/hood/assault
```

Comment:
you boofed it

Reply:
scar doesn't realize i removed that on purpose